Python compatibility issue in Dockerfile

When following instructions for how to build and view ODK docs locally using Docker, I saw the following error message:

```
sphinx-autobuild --poll -p 8080 -H 0.0.0.0 src build
Traceback (most recent call last):
  File "/usr/local/bin/sphinx-autobuild", line 5, in <module>
    from sphinx_autobuild import main
  File "/usr/local/lib/python3.10/site-packages/sphinx_autobuild/__init__.py", line 22, in <module>
    from livereload import Server
  File "/usr/local/lib/python3.10/site-packages/livereload/__init__.py", line 15, in <module>
    from .server import Server, shell
  File "/usr/local/lib/python3.10/site-packages/livereload/server.py", line 20, in <module>
    from tornado.wsgi import WSGIContainer
  File "/usr/local/lib/python3.10/site-packages/tornado/wsgi.py", line 40, in <module>
    from tornado import httputil
  File "/usr/local/lib/python3.10/site-packages/tornado/httputil.py", line 107, in <module>
    class HTTPHeaders(collections.MutableMapping):
AttributeError: module 'collections' has no attribute 'MutableMapping'
make: *** [Makefile:19: autobuild] Error 1
```

[This article](https://stackoverflow.com/questions/69464512/django-rest-error-attributeerror-module-collections-has-no-attribute-mutab) suggests that some of the code is not compatible with python version 3.10, which is the version off of which Docker image is currently built.

I pinned the base Docker image to 3.9.7, which seemed to fix the issue.

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,4 +1,4 @@
-FROM python:3
+FROM python:3.9.7
 
 WORKDIR /mnt
 

